README.md

```diff
@@ -269,6 +269,12 @@ You can select your own combination of Containers form the list below:
 docker-compose exec workspace bash
 ```
 <br />
+<br>
+2a - Alternatively, Windows Powershell users can execute the following command to enter a running container.
+```bash
+docker exec -it {container-id} bash
```

Comment:
Why not using docker-compose? you can install Docker Compose on Windows!! and accessing the containers using Docker directly is also always an option.